news/2018-06-03-project-loved-week-of-june-3rd.md

```diff
@@ -0,0 +1,254 @@
+
+## <a name="osutaiko" id="osutaiko"></a>osu!taiko
+
+[![](/wiki/shared/news/2018-06-03-project-loved-week-of-june-3rd/taiko/boogie-woogie-splatter-show.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[t+pazolite - Boogie Woogie Splatter Show](https://osu.ppy.sh/beatmapsets/52068) by [12358](https://osu.ppy.sh/users/1238439)  
+*written by [Gezoda](https://osu.ppy.sh/users/481582)*
+
+Time to hop into the time machine to go back in 2012, with 12358's first osu!taiko beatmap to enter the polls, Boogie Woogie Splatter Show! Don't be fooled by the fairly simplistic style coming with this, because this monster really packs a punch! Picked by the mappool selectors for the semifinals of the osu!taiko World Cup 2017, the first two thirds of the beatmap will be a cakewalk for many people, but once the final kiai kicks in, an unforgiving stream mixing 1/4 with 1/6 will kill anyone unprepared for it, and the low OD will shatter any hope of passing the map.
+
+[![](/wiki/shared/news/2018-06-03-project-loved-week-of-june-3rd/taiko/nijibishox1hadairosenjosenjo.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[tarolabo - NijiBishox1HadairoSenjoSenjo](https://osu.ppy.sh/beatmapsets/328741) by [Nyan](https://osu.ppy.sh/users/36102)  
+*written by [Backfire](https://osu.ppy.sh/users/263110)*
+
+This particular set has been brought back from the dead, just barely missing out on the then 80% threshold to be loved before. As our system has changed since then, we want to introduce a map that has been denied every now and then to give them another chance at being loved!
+
+Beautifully mapped and well respected in the community, this set is a great introduction to the world of Nyan and tarolabo maps. The two have hooked up a lot in the past, and they're a very well known couple within the taiko community. Nyan's abstract mapping compliments the abrasive style of music tarolabo makes, and with a full spread of difficulties, there should be something here for everyone.
+
+[![](/wiki/shared/news/2018-06-03-project-loved-week-of-june-3rd/taiko/tano-c-tour-2012-the-anthem.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[t+pazolite - TANO\*C TOUR 2012 the Anthem](https://osu.ppy.sh/beatmapsets/179223) by [Fraolinch](https://osu.ppy.sh/users/205257)  
+*written by [Gezoda](https://osu.ppy.sh/users/481582)*
+
+Let's give a warm welcome to Fraolinch, as it's their first time in the pollings! And to start things off for them, here's TANO\*C TOUR 2012 the Anthem, a great yet underrated beatmap set to the tune of one of the remixes of the original Anthem songs. If you love taiko, t+pazolite and difficult maps testing many skill sets, this is a must-play, and also a good introduction to their maps. It really shows that Fraolinch had a blast mapping this map, so it's definitely worth giving a try.
```

Comment:
`love taiko` -> `love osu!taiko`

Reply:
I'm going to keep this one as-is because it's referring to the kind of game in general, not really osu!

changed everything else